src/Raven.Server/Config/Categories/ServerConfiguration.cs

```diff
@@ -1,11 +1,18 @@
 using System.ComponentModel;
 using Raven.Server.Config.Attributes;
 using Raven.Server.Config.Settings;
+using Sparrow.Platform;
+using Sparrow.Utils;
 
 namespace Raven.Server.Config.Categories
 {
     public class ServerConfiguration : ConfigurationCategory
     {
+        public ServerConfiguration()
+        {
+            ThreadPoolMinThreads = (PlatformDetails.Is32Bits ? 2 : 4) * ProcessorInfo.ProcessorCount;
```

Comment:
IIRC, that would actually reduce the number of small machines, no?
The default is ~25 ?

Reply:
No. The default is the number of processors on a system. 

https://docs.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads?view=netcore-3.1